### PR TITLE
Management access error

### PR DESCRIPTION
**Pull request by Starkandco**

### CODE REVIEW

File: client/src/app/login/login.component.html [hunk 1]
```diff
@@ -31,5 +32,10 @@ <h1>{{ isManagementLogin ? "Management Login" : "Public Login" }}</h1>
       <button mat-button>Cancel</button>
       <button mat-button (click)="login()">Login</button>
     </div>
+    <div class="centered">
```
Comment (grizzle2101) [inline client/src/app/login/login.component.html:35]: Heyo, bit of a no no to put styling directly into HTML, makes it a nightmare when trying to debug styles later.
You can use mat error, just between the password & buttons like so.

>       "<mat-error class="centered">{{ errorText }}</mat-error>"

File: backend/src/interfaces/user.interface.ts [hunk 2]
```diff
@@ -3,4 +3,5 @@ export interface User {
   name: string;
   email: string;
   password: string;
+  isManagement: boolean;
```
Comment (grizzle2101) [inline backend/src/interfaces/user.interface.ts:6]: I have just updated the DB schema to go along with your updates. You might want to pull and wipe your DB to get them applied.
Also a tip if you're ever locked out keep a postman collection around
{
    "name": "conor",
    "email": "conor@gmail.com",
    "password": "conor",
    "isManagement": true
}

**Review by grizzle2101 — CHANGES_REQUESTED** [review]
Can you update the login HTML when you get the chance.
Fair play on the PR, tested & running well!

File: client/src/app/login/login.component.html (hunk 1 above)
Comment (Starkandco) [reply client/src/app/login/login.component.html]: Think I've got this right now?

**Review by grizzle2101 — APPROVED** [review]

**Pull request merged**